test/unit_test.vcxproj.filters

```diff
@@ -45,5 +45,8 @@
     <ClCompile Include="player_locations_test.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
+    <ClCompile Include="jack_move_test.cpp">
+      <Filter>Source Files</Filter>
+    </ClCompile>
```

Comment:
These lines tell Visual Studio how to display the file in the Solution Explorer. In this case, it sorts the file under `Source Files`